src/contract_wrappers/contract_wrapper.ts

```diff
@@ -16,4 +49,10 @@ export class ContractWrapper {
             await this._web3Wrapper.getContractInstanceFromArtifactAsync<A>(artifact, addressIfExists);
         return contractInstance;
     }
+    protected _getEventSignatureFromAbiByName(abi: Web3.ContractAbi, eventName: ContractEvents): string {
+        const eventAbi = _.filter(abi, {name: eventName})[0] as Web3.EventAbi;
```

Comment:
Instead of `_.filter(abi, {name: eventName})[0]`, use `_.find(abi, {name: eventName})` it only returns the first one it finds.